pypesto/objective/aggregated.py

```diff
@@ -189,6 +189,11 @@ def aggregate_hess(self, x):
     def aggregate_hessp(self, x):
         return sum(objective.hessp(x) for objective in self.objectives)
 
+    def reset_steadystate_guesses(self):
```

Comment:
not nice but should work. steady states are only for AmiciObjective, right? could you add a comment here?

Reply:
done